internal/gapicgen/generator/genproto.go

```diff
@@ -133,10 +133,13 @@ func (g *GenprotoGenerator) Regen(ctx context.Context) error {
 		grpc := !noGRPC[pkg]
 		pk := pkg
 		fn := fileNames
-		grp.Go(func() error {
-			log.Println("running protoc on", pk)
-			return g.protoc(fn, grpc)
-		})
+
+		if !isMigrated(pkg) {
```

Comment:
What do you think about also logging a statement for the `isMigrated` `true` case, to indicate that `protoc` is not running due to aliasgen.

Reply:
done.